lib/Edited/prototype/discern.js

```diff
@@ -2,10 +2,21 @@
 
 module.exports = function (type) {
   var self = this
-  self.callbackAny()
+  if (type !== 'click' && type !== 'arrow-key') {
+    // click only serves to indicate that
+    // a selection change probably occurs next.
+    // keeping track of selection is much more complex
+    // so we just assume that selection was changed
+    self.callbackAny()
+  }
   self.previousEditType = self.lastEditType
   self.lastEditType = type
   if (!self.previousEditType) return
+  if (type === 'click' || type === 'arrow-key') {
+    // there was likely no edit
+    // ―only selection change
```

Comment:
Let me see if I follow this logic. Now, clicks and arrow-keys set the `lastEditType`, which means that if a subsequent event is anything else (e.g. a keypress), that keypress will start a new sensibleEdit. Is that a fair summary?